.github/workflows/docker.yml

```diff
@@ -38,3 +38,21 @@ jobs:
         push: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
+    - id: meta-dev
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/projectlighthousecau/luna
+        tags: |
+          type=raw,value=dev,enable={{is_default_branch}}
+          type=sha,format=long
+    - name: Build and Push dev image
+      uses: docker/build-push-action@v5
```

Comment:
Instead of running these sequentially, I'd suggest running them either as parallel jobs, maybe via different matrix configurations. This would avoid a fair bit of duplication here.

Reply:
I thought that there might be a way to avoid code duplication, but I didn't know how, so I simply duplicated it.
Feel free to refactor :smile: